client/src/plugins/libparsec/definitions.ts

```diff
@@ -436,6 +436,18 @@ export type ClientCancelInvitationError =
   | ClientCancelInvitationErrorNotFound
   | ClientCancelInvitationErrorOffline
 
+// ClientChangeAccessError
+export enum ClientChangeAccessErrorTag {
+    ChangeAccess = 'ClientChangeAccessErrorChangeAccess',
```

Comment:
It's lacking in details. `clientStart` can have all these errors
```
ClientStartErrorInternal
 ClientStartErrorLoadDeviceDecryptionFailed
ClientStartErrorLoadDeviceInvalidData
ClientStartErrorLoadDeviceInvalidPath
```
which probably means that `clientChangeAccess` should at least have those because you get the same problems (finding the device key file, decrypting it, deserializing it).

Reply:
Done